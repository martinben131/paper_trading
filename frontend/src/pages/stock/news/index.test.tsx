import React from "react";
import { shallow } from "enzyme";
import Register from "./index";

jest.mock("../store", () => ({
  useProfile: () => [{}],
  useNews: () => [
    {
      code: "MSFT",
      loading: false,
      data: [],
    },
  ],
}));

jest.mock("react-router-dom", () => ({
  useParams: () => ({
    code: "MSFT",
  }),
}));

test("Renders news page correctly", () => {
  const component = shallow(<Register />);
  expect(component).toMatchSnapshot();
});
