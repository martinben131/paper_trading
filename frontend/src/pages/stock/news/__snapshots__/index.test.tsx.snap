// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders news page correctly 1`] = `
<Fragment>
  <div
    className="newsTitle"
  >
    News Articles
  </div>
</Fragment>
`;
